pyscriptjs/src/remote_interpreter.ts

```diff
@@ -72,9 +85,35 @@ export class RemoteInterpreter extends Object {
     // TODO: Remove this once `runtimes` is removed!
     interpreter: InterpreterInterface & ProxyMarked;
 
-    constructor(src = 'https://cdn.jsdelivr.net/pyodide/v0.23.2/full/pyodide.js') {
+    pyodide: Promise<PyodideHelper> | null;
+
+    constructor(src = 'https://cdn.jsdelivr.net/pyodide/v0.23.2/full/pyodide.js', bootstrap = false) {
         super();
         this.src = src;
+        this.pyodide = bootstrap
+            ? new Promise(resolve => {
+                  const pyConfig = document.querySelector('py-config');
+                  const config = pyConfig?.getAttribute('src') || pyConfig?.textContent;
+                  // register all default plugins
+                  registerPlugin(
```

Comment:
this is a good question ... there's some confusion around plugins in *pyscript.core* because there are **two kind of plugins**:

  1. those live on the DOM, like **core plugins** you register via *core* indeed
  2. the legacy understanding of a plugin ... which is all about **py-script** plugins, not about *core* as intended in here

In short, the *py-script* plugin will have its own *plugins* architecture, like any other *core* plugin could ... the *py-script* plugin architecture doesn't get to ditch what other DOM/core plugins should do with ... well ... plugins, it's just using whatever mechanism it had before.

As it's clear everyone but me is confused by this, should I rename the `registerPlugin` to `registerDOMPlugin` ???

/cc @antocuni @hoodmane @fpliger